core/gdxsv/gdxsv_gui_settings.cpp

```diff
@@ -32,6 +32,49 @@ static const char* t(std::initializer_list<const char*> texts) {
 	return *(texts.begin() + index);
 }
 
+void gdxsv_apply_base_settings()
+{
+	// Frame Limit
+	config::LimitFPS = false;
```

Comment:
Plase set LimitFPS true. bacause enabled automatically during GGPO matches.

Reply:
Since both `LimitFPS` and `FixedFrequency` are overridden during GGPO, so we should skip both settings?

Reply:
only LimitFPS.
Disableing FixedFrequency causes unstable fps in offline play.

Reply:
Done.
I guess macOS doesn't have the audio glitch introduced by disabling AudioSync 😄